bin/generate-version-file.js

```diff
@@ -0,0 +1,57 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+// @flow
+
+/* This script generates the version file, as requested by the dockerflow
+ * requirements described in
+ * https://github.com/mozilla-services/Dockerflow/blob/master/docs/version_object.md.
+ * It uses the environment variables set by CircleCI.
+ */
+
+const fs = require('fs');
+const path = require('path');
+
+function checkEnvironment() {
+  const isRunningInCircleCi = process.env.CIRCLECI === 'true';
+  if (!isRunningInCircleCi) {
+    throw new Error(
+      'This script will not work correctly outside of a CircleCI environment.'
+    );
+  }
+
+  const isAtGitRoot = fs.existsSync('.git');
+  if (!isAtGitRoot) {
+    throw new Error('This script must be run at the root of the repository.');
+  }
+}
+
+function writeVersionFile() {
+  const repositoryUrl = process.env.CIRCLE_REPOSITORY_URL;
+  const commitHash = process.env.CIRCLE_SHA1;
+  const buildUrl = process.env.CIRCLE_BUILD_URL;
+  const version = '1.0';
```

Comment:
I considered fetching this from package.json, but then settled for the simpler path. Please tell me what you think!

Reply:
I'm not really understanding what this version means exactly. Is this the version of our app? Is it of the container? When does this update?

Generally, I'd prefer to keep version numbers in more "declaration"-like places. I think you can use `require` on `json` files. `require('../package.json').version`.

Reply:
I don't really know what the version means either... should it increase each time we deploy? Or is it just informational (and in that case we don't really have a use for it). That's why I kept it here for now.
I asked our contact at devops for some more context about this value.

Reply:
From the doc in https://github.com/mozilla-services/Dockerflow/blob/master/docs/version_object.md:

*  The release version of the application
   *  can be left blank for non-release versions

So I think I'll do that for now, and revisit when we'll need something better.